### DISCUSSION TITLE
Bochs 2.8 release with SMP support

### DISCUSSION THREAD

**Discussion by ghost** [bochs-emu/Bochs#328] [Ideas]
When i booted my vm with smp option on Bochs 2.8 it wont boot.
So that means that Bochs 2.8 has been not compiled with SMP support.

**Comment by Vort** [comment]
That's how Bochs is built by default.
Why do you need SMP option?

**Comment by ghost** [comment]
Bochs 2.6 had that option why you stopped making smp releases

**Comment by ghost** [comment]
I know that bochs supports that feature

**Reply by stlintel** [reply]
Do you have a problem to recompile by yourself ?

**Reply by ghost** [reply]
Sorry but i dont understand compiling. And where i would find it (the release of bochs).

**Reply by ghost** [reply]
And how to compile it

**Comment by bachimusprime** [comment]
This user is a known GitHub spammer and has already been banned from the 86Box GitHub for spamming their repos with dumb asinine feature requests in their discussions tab.

**Comment by Vort** [comment]
I will put SMP binary here just in case: [bochs_smp.zip](https://github.com/user-attachments/files/15956811/bochs_smp.zip).
